.buildkite/scripts/common.sh

```diff
@@ -501,4 +508,9 @@ if are_paths_changed "${packaging_changeset[@]}" ; then
   export PACKAGING_CHANGES="true"
 fi
 
+if [[ "$BUILDKITE_STEP_KEY" == "xpack-winlogbeat-pipeline" || "$BUILDKITE_STEP_KEY" == "xpack-metricbeat-pipeline" || "$BUILDKITE_STEP_KEY" == "metricbeat-pipeline" ]]; then
+  # Set the MODULE env variable if possible
+  defineModuleFromTheChangeSet "${BEATS_PROJECT_NAME}"
```

Comment:
why? A comment in the code can help to understand the rationale 🙏

Reply:
The variable MODULE should be defined for different pipelines and steps as well. That's why I moved that to the common script.
Updated the comment.